cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -259,6 +261,24 @@ func (k *Kuberhealthy) runCheck(stopChan chan bool, c KuberhealthyCheck) {
 		details := health.NewCheckDetails()
 		details.Namespace = c.CheckNamespace()
 		details.OK, details.Errors = c.CurrentStatus()
+
+		if k.MetricForwarder != nil {
+			checkStatus := 0
```

Comment:
var checkStatus also works here and is more normal. no need to change this, though